model/Core/Properties/elementValue.md

```diff
@@ -0,0 +1,20 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# elementValue
+
+## Summary
+
+A value used in a generic key-value pair that refers to an Element
```

Comment:
```suggestion
A value used in a key-value pair with a generic key that refers to an Element
```

Reply:
With this, we may also like to update the summary and description of the `key` property.

`key`'s current summary is "A key used in a generic key-value pair."

`key`'s current description is "A key used in generic a key-value pair." (note the "generic a")

https://github.com/spdx/spdx-3-model/blob/develop/model/Core/Properties/key.md